api/bases/octavia.openstack.org_octaviaamphoracontrollers.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
it seems you are using an old controller-gen version. remove the `bin/controller-gen` and rerun the make generate manifests

Reply:
Thanks!